cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-default-state.cy.js

```diff
@@ -0,0 +1,109 @@
+import Permissions from '../../../support/dictionary/permissions';
+import BulkEditSearchPane from '../../../support/fragments/bulk-edit/bulk-edit-search-pane';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import TopMenu from '../../../support/fragments/topMenu';
+import Users from '../../../support/fragments/users/users';
+import FileManager from '../../../support/utils/fileManager';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import TopMenuNavigation from '../../../support/fragments/topMenuNavigation';
+
+let user;
+const item = {
+  instanceName: `testBulkEdit_${getRandomPostfix()}`,
+  itemBarcode: getRandomPostfix(),
+};
+const holdingsHRIDFileName = `holdingsHRIDFileName${getRandomPostfix()}.csv`;
+
+describe('bulk-edit', () => {
+  describe('in-app approach', () => {
+    before('Create test data', () => {
+      cy.createTempUser([
+        Permissions.bulkEditView.gui,
+        Permissions.bulkEditEdit.gui,
+        Permissions.uiInventoryViewCreateEditHoldings.gui,
+      ]).then((userProperties) => {
+        user = userProperties;
+        item.instanceId = InventoryInstances.createInstanceViaApi(
+          item.instanceName,
+          item.itemBarcode,
+        );
+        cy.getHoldings({
+          limit: 1,
+          query: `"instanceId"="${item.instanceId}"`,
+        }).then((holdings) => {
+          item.holdingHRID = holdings[0].hrid;
+          FileManager.createFile(`cypress/fixtures/${holdingsHRIDFileName}`, holdings[0].hrid);
+        });
+        cy.login(user.username, user.password, {
+          path: TopMenu.bulkEditPath,
+          waiter: BulkEditSearchPane.waitLoading,
+        });
+      });
+    });
+
+    after('Delete test data', () => {
+      Users.deleteViaApi(user.userId);
+      InventoryInstances.deleteInstanceAndHoldingRecordAndAllItemsViaApi(item.itemBarcode);
+      FileManager.deleteFile(`cypress/fixtures/${holdingsHRIDFileName}`);
+    });
+
+    it(
+      'C375095 Verify default state of Bulk edit landing page (In app - inventory approach) (firebird) (TaaS)',
+      { tags: ['extendedPath', 'firebird'] },
+      () => {
+        BulkEditSearchPane.verifyPanesBeforeImport();
+        BulkEditSearchPane.verifySetCriteriaPaneSpecificTabs('Identifier');
+        BulkEditSearchPane.verifySpecificTabHighlighted('Identifier');
+        BulkEditSearchPane.verifyRecordIdentifierEmpty();
```

Comment:
are you checking for alphabetical order?
![image](https://github.com/folio-org/stripes-testing/assets/72025248/6aa4507f-9705-48e8-8e8c-fb96b1a1e761)

Reply:
No, but I`ll check!